core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/parse.kt

```diff
@@ -64,8 +67,17 @@ public data class ParserOptions(
     val dateTimeFormatter: DateTimeFormatter? = null,
     val dateTimePattern: String? = null,
     val nullStrings: Set<String>? = null,
+    val skipTypes: Set<KType> = emptySet(),
```

Comment:
Maybe `skipTypes` be better be inverted to `checkTypes`. But that would mean exposing all `Parsers` so users/csv readers can filter which types to consider when parsing and which to skip. WDYT?

Reply:
Really weird feature, let's live a little bit with that solution to reflect later in code

Reply:
It's mostly a performance thing, but the problem is that we don't have a user-facing way to refer to parsers, so we cannot say: "please parse this string column and try this type but not this type". I'm not sure how the API should look here